engine/profiler/src/profile_render.cpp

```diff
@@ -86,7 +86,7 @@ namespace dmProfileRender
     static const int SAMPLE_FRAMES_COUNT_WIDTH = 3 * CHARACTER_WIDTH;
 
     static const int COUNTERS_NAME_WIDTH  = 32 * CHARACTER_WIDTH;
-    static const int COUNTERS_COUNT_WIDTH = 8 * CHARACTER_WIDTH;
+    static const int COUNTERS_COUNT_WIDTH = 12 * CHARACTER_WIDTH;
```

Comment:
<img width="184" alt="image" src="https://github.com/defold/defold/assets/2209596/ceac2f42-baf3-44ac-add0-9018614a5df2">
Make sure we have enough space

Reply:
An option is to use `GuiDynamicTextureSizeKb`and divide by 1024. Or use Mb even

Reply:
I like it! Done!